doc/htmldoc/synapses/handling_connections.rst

```diff
@@ -1,7 +1,10 @@
 .. _handling_connections:
 
-How to handle connections (synapses)
-=====================================
+How to handle created connections
+=================================
+
+Here, we describe how to inspect and modify already created connections.
+For creating new connections, refer to :ref:`connection_management`, and for parameterizing synapses, see :ref:`synapse_spec`.
```

Comment:
The parameterization you mean here is the one during creation, isn't it? Then the sentence is confusing. Maybe something like
```suggestion
:ref:`connection_management` describes how to create new connections and :ref:`synapse_spec` how to parameterize synapses as they are created.
```